docs/docs/contribute/getting-started.md

```diff
@@ -14,109 +14,193 @@ license: CC-BY-SA-4.0
 
 Learn how to set up the development environment to start working with our projects!
 
-## Code Editor ##
 
-A code editor allows you to work with code more easily. We will be using Visual Studio Code (VS Code) for this tutorial. You can download it [here](https://code.visualstudio.com/download), and install it with the default settings.
+## Setting up the C/C++ Compiler
 
-!!! note
-    You are free to use any code editor of your choice, although you will have to complete steps of this tutorial through alternate means.
+A compiler translates human-readable code into machine code. We will be setting up the C/C++ compiler so that our code written in the human-readable C++ language can be understood by the computers!
 
-## Setting up the GCC Compiler ##
+=== "Windows"
 
-A compiler translates human-readable code into machine code. We will be setting up the GCC compiler so that our code written in the human-readable C++ language can be understood by the computers!
+    First, [download Chocolatey](https://docs.chocolatey.org/en-us/choco/setup#installing-chocolatey) and install it using an [administrative shell](https://www.howtogeek.com/194041/how-to-open-the-command-prompt-as-administrator-in-windows-8.1/). Chocolatey is a package manager that simplifies the installation of the tools we will need.
 
-First, [download MSYS2](https://www.msys2.org/) and install it with the default settings. MSYS2 helps to set up MinGW-w64, which uses the GCC compiler to create Windows programs.
+    We will now install the compiler and other build tools using Chocolatey.
+    Open up the Command Prompt and enter the following commands:
 
-We will now install the Mingw-w64 toolset which includes GCC, and other build tools that will be employed later. When the MinGW terminal pops up, enter the following command:
+    ??? tip
```

Comment:
This tip should appear after the the commands as it breaks the instructional flow from "commands:" to the commands.

Reply:
I thought about that but given that we also have a note below the command it didn't feel right to me. I'm unsure for that one, I can't find a good place to have it.